src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,72 @@
 'use strict';
 
+const NAME = '';
```

Comment:
The constant 'NAME' is defined but never used. If you don't plan to use it, consider removing it to keep the code clean.